core/src/main/java/org/pentaho/di/connections/vfs/VFSDetailsComposite.java

```diff
@@ -33,4 +33,21 @@ public interface VFSDetailsComposite {
   default String validate() {
     return null;
   }
+
+  /**
+   * Gets the Object of SelectionAdapterFileDialogTextVar for the browse file dialog.
+   * @return Object of SelectionAdapterFileDialogTextVar or null otherwise.
```

Comment:
Should document the parameter as TextVar.

Reply:
Addressed